src/EventService.Business/Commands/EventCategory/CreateEventCategoryCommand.cs

```diff
@@ -41,26 +42,28 @@ public class CreateEventCategoryCommand: ICreateEventCategoryCommand
     _responseCreator = responseCreator;
     _contextAccessor = contextAccessor;
   }
-  public async Task<OperationResultResponse<Guid?>> ExecuteAsync(CreateEventCategoryRequest request)
+  public async Task<OperationResultResponse<bool>> ExecuteAsync(CreateEventCategoryRequest request)
   {
-    if (!await _accessValidator.HasRightsAsync(Rights.AddEditRemoveCompanyData))
+    if (!await _accessValidator.HasRightsAsync(Rights.AddEditRemoveUsers))
```

Comment:
Not sure, but here is has to be some right about event or category @vorobevamv

Reply:
it's OK. Rights for EventService: AddEditRemoveUsers (it's HRs) and/or admin